operator/scripts/inject-custom-image.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+set -euxo pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+VERSION="$1"
+VALUE="$2"
+
+yq ea -i ".spec.install.spec.deployments[0].spec.template.spec.containers[0].env += {\"name\": \"OPERATOR_KEYCLOAK_IMAGE\", \"value\": \"$VALUE\"}" $SCRIPT_DIR/../olm/$VERSION/manifests/keycloak-operator.v$VERSION.clusterserviceversion.yaml
```

Comment:
No need for this.

Reply:
if the configuration through the subscription works as expect I'll remove the file entirely

Reply:
removed